cfgov/jinja2/v1/ask-cfpb/category-page.html

```diff
@@ -39,43 +39,40 @@
         </div>
     </section>
 
-    <section class="ask-categories">
+    {% if choices|length > 0 %}
+    <section class="ask-categories
+                    block block__bg 
+                    block__flush-sides 
+                    block__flush-bottom 
+                    block__bg-extend-right">
+        <h3>What is your question about?</h3>
+        <nav class="content-l subcategory-nav">
+            {% for subcategory in choices %}
+            {% if loop.index == 1 %}
+            <ul class="content-l_col content-l_col-1-2 facets_list list list__unstyled ">
+            {% endif %}
+            {% set count = choices|length %}
+            {% if loop.index == ( count // 2 ) + ( 2 if count % 2 else 1) %}
```

Comment:
This approach was unnecessarily complicated, so I switched to using the `slices` filter to create columns in https://github.com/cfpb/cfgov-refresh/pull/3033/commits/21c18afdc25776df21caf4cf90134ef9718838e6. Thanks for calling this out!